### PR TITLE
Building road dist

### PR DESCRIPTION
**Pull request by j3r3m1**

### CODE REVIEW

**Comment by j3r3m1** [comment]
About this indicator and the building distance one, will it be possible to set a default value for some of the input parameters used in the IProcess ? For these specific IProcesses, the bufferDist param should be set by default to 100 m.

**Comment by ebocher** [comment]
You can initialize a distance with a default value in the closure. Check if the input distance is greater than the default value or less than a specific value then use the default one.

**Comment by j3r3m1** [comment]
OK, I have set the default value. I do not think it is necessary to define any condition (lower than the default), do you ?

File: geoindicators/src/main/groovy/org/orbisgis/BuildingIndicators.groovy [hunk 1]
```diff
@@ -239,7 +239,7 @@ static IProcess buildingMinimumBuildingSpacing() {
             [inputBuildingTableName: String,inputFields:String[],bufferDist: Double
              , outputTableName: String, datasource: JdbcDataSource],
             [outputTableName : String],
-            { inputBuildingTableName,inputFields, bufferDist, outputTableName, JdbcDataSource datasource ->
+            { inputBuildingTableName,inputFields, bufferDist = 100, outputTableName, datasource ->
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/BuildingIndicators.groovy]: +1

**Comment by ebocher** [comment]
Brilliant

**Pull request merged**